### ISSUE TITLE
Organization "Linked domains" label is broken

### ISSUE THREAD

**Issue by marien-probesys** [Probesys/bileto#948]
## Problem

The label "Linked domains" in the organization form has no `for` attribute. Thus, the input has no label.

This is because the `domains` field is a CollectionType and that the label cannot apply to a single input in normal situation. However, this CollectionType is rendered with the `form/_input_texts.html.twig` template, which displays a single input. The label must be attached to this one.

## Solution

Make sure that the label has a `for` attribute.

## Specifications

First, rename the component into `multitext`.

Also, move the code of `templates/form/_input_texts.html.twig` into the form theme (`templates/form/bileto_theme.html.twig`).

Then, make sure that the label of this new component generate a `for` attribute corresponding to the input id.

Finally, set `block_prefix = 'multitext'` to the `domains` field of the Organization form.
